src/core/core.interaction.js

```diff
@@ -97,6 +97,10 @@ function getIntersectItems(chart, position, axis, useFinalPosition) {
   }
 
   const evaluationFunc = function(element, datasetIndex, index) {
+    const chartArea = chart.chartArea;
+    if (element.x < chartArea.left || chartArea.right < element.x || element.y < chartArea.top || chartArea.bottom < element.y) {
```

Comment:
I would use the `_isPointInArea` helper function here.

Reply:
This seems to be the only issue here, @t-mangoe will you be able to update this PR?

Reply:
Oh, sorry. I noticed the comment, but have not yet updated it.
I will try to respond, so I hope you can wait.
(Or, I allow edits by maintainers, so it is ok that maintainers update this PR.)